app/jobs/send_application_job.rb

```diff
@@ -30,22 +31,19 @@ def send_pdf
   def fax_pdf
     if fax_phone_number_exists?
       Fax.send_fax(
-        number: fax_number,
+        number: fax_recipient_number,
         file: complete_form_with_cover.path,
         recipient: fax_recipient_name,
       )
     end
   end
 
-  def fax_number
-    ENV.fetch("FAX_NUMBER", "")
-  end
-
-  def fax_recipient_name
-    ENV.fetch("FAX_RECIPIENT", "Michigan Bridges")
+  def fax_recipient
+    FaxRecipient.new(residential_address: residential_address)
   end
+  delegate :number, :name, to: :fax_recipient, prefix: true
```

Comment:
this is not highly important to me, but I generally prefer explicit methods rather than `delegate` because when looking for the logic behind something I usually search for the method name, eg `def fax_recipient_number` and if it is defined with`delegate` I will not find it.